distribution/docs/src/main/resources/_contents/_broker/securing-artemis-contents.adoc

```diff
@@ -0,0 +1,124 @@
+==== Introduction
+
+This section describes how to configure ${branding} and Artemis to perform authentication and
+authorization against an LDAP server.
+
+Artemis provides the ability to apply role-based security to queues based on addresses
+(see https://activemq.apache.org/artemis/docs/${artemis.version}/security.html[the Artemis documentation]
+for details). It can be configured to use an LDAP server to perform authentication and authorization
+for users who connect to it.
+
+IMPORTANT: If you are setting up multiple ${branding} instances in a cluster for high availability,
+then you will need to perform these steps on each instance.
+
+The ${ddf-security} STS LDAP Login and ${ddf-security} STS LDAP Claims Handler bundles are responsible
+for authenticating and authorizing users with your LDAP server. To configure them for your LDAP server,
+follow the instructions in <<${ddf-security} STS LDAP Login,${ddf-security} STS LDAP Login>> and
+<<${ddf-security} STS LDAP Claims Handler,${ddf-security} STS LDAP Claims Handler>>.
+
+include::${project.build.directory}/doc-contents/_ddf/docs-${ddf.version}/_security-framework/security-sts-ldap-login-contents.adoc[]
```

Comment:
These can be replaced with links to the corresponding sections in the DDF documentation so we don't include them twice.